web/app/test/VersionTest.jsx

```diff
@@ -15,14 +18,15 @@ describe('Version', () => {
   let component, fetchStub;
 
   function withPromise(fn) {
-    return component.find("Version").get(0).serverPromise.then(fn);
+    return component.find("Version").instance().serverPromise.then(fn);
   }
 
   beforeEach(() => {
     fetchStub = sinon.stub(window, 'fetch');
   });
 
   afterEach(() => {
+    component = null;
```

Comment:
curious: what's the motivation here?

Reply:
ah! since we're sharing the component variable in all the tests, I just wanted to be sure we were resetting it after each use.